.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "third_party/brotli/research/esaxx"]
```

Comment:
@kaidokert I'm not sure I'm doing the right thing here. As the path here should be the path to root dir, if I don't add this file, all tests fail for no url for this submodule. But all third_party code have .gitmodules file in the sub dir under third_party. Do you know how do usually we deal with this case?

Reply:
Sorry to hold this up - i've gone ahead and cloned your setup separately in sandbox to investigate _why_ this is happening, it should not. ( https://github.com/youtube/cobalt_sandbox/pull/437 )
 
It looks like there's another patch missing from the checkout actions code. Meanwhile - feel free to restore it in a state so that checks run, i'll separately try to resolve this issue in the sandbox

Reply:
Ok i found whats wrong, PR up at #406. You can cherry-pick that fix in to the PR if rebase is too much overhead.